.gitignore

```diff
@@ -1 +1,3 @@
 /target
+
+.idea/
```

Comment:
Please don't add IDE specific files to the project. You should configure a global git ignore list for the tools you use locally :)